lib/travis/build/script.rb

```diff
@@ -45,6 +45,10 @@ module Travis
   module Build
     class Script
       TEMPLATES_PATH = File.expand_path('../templates', __FILE__)
+      INTERNAL_RUBY_REGEX = ENV.fetch(
+        'TRAVIS_INTERNAL_RUBY_REGEX',
+        'ruby-(2\.[0-2]\.[0-9]|1\.9\.3)'
```

Comment:
This regex should maybe be `^ruby-...`, although there are no known other versions that I think of that contain the ruby-2.? or ruby-1.9.3 substring.